dev-go/fuzzy/fuzzy-0.0.3.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+EGO_PN=github.com/sahilm/fuzzy
+
+if [[ ${PV} = *9999* ]]; then
+	inherit golang-vcs
+else
+	KEYWORDS="~amd64"
+	SRC_URI="https://${EGO_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	inherit golang-vcs-snapshot
+fi
+inherit golang-build
+
+DESCRIPTION="Go library that provides fuzzy string matching"
+HOMEPAGE="https://github.com/sahilm/fuzzy"
+LICENSE="MIT"
+SLOT="0/${PVR}"
+RDEPEND=""
```

Comment:
```
>>> Test phase: dev-go/fuzzy-0.0.3
env GOPATH=/tmp/portage/dev-go/fuzzy-0.0.3/work/fuzzy-0.0.3:/usr/lib/go-gentoo go test -v -work -x github.com/sahilm/fuzzy
WORK=/tmp/portage/dev-go/fuzzy-0.0.3/temp/go-build264483519
# github.com/sahilm/fuzzy
src/github.com/sahilm/fuzzy/fuzzy_test.go:12:2: cannot find package "github.com/kylelemons/godebug/pretty" in any of:
	/usr/lib/go/src/github.com/kylelemons/godebug/pretty (from $GOROOT)
	/tmp/portage/dev-go/fuzzy-0.0.3/work/fuzzy-0.0.3/src/github.com/kylelemons/godebug/pretty (from $GOPATH)
	/usr/lib/go-gentoo/src/github.com/kylelemons/godebug/pretty
FAIL	github.com/sahilm/fuzzy [setup failed]

```

Reply:
Oof. Alright, I'll add that test-phase dependency.